packages/@react-spectrum/searchfield/docs/SearchField.mdx

```diff
@@ -0,0 +1,174 @@
+```
+
+Placeholder text that describes the expected value or formatting for the SearchField can be provided using the `placeholder` prop. 
+Placeholder text will only appear when the SearchField is empty, and should not be used as a substitute for labeling the component with a visible label.
+
+```tsx example
+<SearchField placeholder="abc@adobe.com" label="Email" />
+```
+
+## Labeling
+
+A visual label should be provided for the SearchField using the `label` prop. If the SearchField is required, the `isRequired` and 
+`necessityIndicator` props can be used to show a required state.
+
+```tsx example
+<SearchField label="Street address" marginEnd="20px" />
+<SearchField label="Street address" isRequired necessityIndicator="icon" marginEnd="20px" />
+<SearchField label="Street address" isRequired necessityIndicator="label" marginEnd="20px" />
+<SearchField label="Street address" necessityIndicator="label" />
+```
+
+### Accessibility
+
+If a visible label isn't specified, an `aria-label` must be provided to the SearchField for 
+accessibility. If the field is labeled by a separate element, an `aria-labelledby` prop must be provided using 
+the `id` of the labeling element instead.
+
+```tsx example
+<label id="label-1">Name</label>
+<SearchField aria-labelledby="label-1" marginStart="10px" />
+```
+
+### Internationalization
+
+In order to internationalize a SearchField, a localized string should be passed to the `label` or `aria-label` prop.
+When the `necessityIndicator` prop is set to `"label"`, a localized string will be provided for `"(required)"` or `"(optional)"` automatically.
+
+```tsx example
+<SearchField label="Street address" isRequired necessityIndicator="label" marginEnd="20px" />
+<SearchField label="Street address" necessityIndicator="label" />
+```
+
+## Events
+
+The most commonly used handlers for events in SearchField are the:
+- `onChange` prop which is triggered whenever the value is edited by the user. 
+- `onSubmit` prop which is triggered whenever the value is submitted by the user. 
+- `onClear` prop which is triggered whenever the value is cleared by the user. 
+
+The example below uses `onChange`, `onSubmit`, and `onClear` to update a separate elements with the text entered into the SearchField.
+```tsx example
+function Example() {
+  let [currentText, setCurrentText] = React.useState();
+  let [submittedText, setSubmittedText] = React.useState();
+
+  return (
+    <div>
+      <SearchField
+        placeholder="Enter some text"
+        onClear={() => setCurrentText('')}
+        onChange={setCurrentText}
+        onSubmit={setSubmittedText}
+        label="Mirror"
+        marginEnd="20px"
+        value={currentText}
+      />
+      <div>Mirrored text: {currentText}</div>
+      <div>Submitted text: {submittedText}</div>
+    </div>
+  );
+}
+```
+
+## Props
+
+<PropTable component={docs.exports.SearchField} links={docs.links} />
+
+## Visual Options
+
+### Quiet 
+
```

Comment:
It seems like we include a descriptive sentence when we don't have a link to the spectrum docs. This is an observation to start a discussion.